src/Jobs/Recruit.Vacancies.Jobs/DomainEvents/Handlers/Provider/SetupProviderHandler.cs

```diff
@@ -13,41 +15,40 @@ public class SetupProviderHandler : DomainEventHandler,  IDomainEventHandler<Set
         private readonly ILogger<SetupProviderHandler> _logger;
         private readonly IJobsVacancyClient _client;
         private readonly IEditVacancyInfoProjectionService _projectionService;
+        private readonly IProviderRelationshipsService _providerRelationshipService;
 
         public SetupProviderHandler(ILogger<SetupProviderHandler> logger, 
             IJobsVacancyClient client, 
```

Comment:
IJobsVacancyClient _client is not being used. Can this be removed from the constructor?

Reply:
Removed